tests/pages/firefox_dnt.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+
+from .base import BasePage
+
+
+class FirefoxDNTPage(BasePage):
```

Comment:
I think this makes sense. I did consider this initially, but wasn't quite sure if this would be the right way to do it. I'll make it so!